force-first-last.php

```diff
@@ -149,15 +181,15 @@ function ffl_settings_page() { ?>
 	</style>
 	<div class="wrap">
 		<div class="stranger_studios_banner">
-			<a class="stranger_studios_logo" title="Stranger Studios" target="_blank" href="https://www.strangerstudios.com/?utm_source=plugin&utm_medium=force-first-last&utm_campaign=homepage"><img src="<?php echo esc_url( plugins_url( 'images/Stranger-Studios.png', FFL_BASENAME ) ); ?>" border="0" alt="Stranger Studios(c) - All Rights Reserved"></a>
+			<a class="stranger_studios_logo" title="<?php esc_html_e( 'Stranger Studios', 'force-first-last' ); ?>" target="_blank" href="https://www.strangerstudios.com/?utm_source=plugin&utm_medium=force-first-last&utm_campaign=homepage"><img src="<?php echo esc_url( plugins_url( 'images/Stranger-Studios.png', FFL_BASENAME ) ); ?>" border="0" alt="<?php esc_html_e( 'Stranger Studios(c) - All Rights Reserved', 'force-first-last' ); ?>"></a>
```

Comment:
Use esc_attr_e() for strings that show up in attributes inside of tags.

This does an extra check for quotes that could break the HTML.

Reply:
Other spots down below too.